### ISSUE TITLE
He, yang jiao shou. Could you please help me out about how to calculate force 

### ISSUE THREAD

**Issue by ztdepztdep** [yjhp1016/taichi_LBM3D#37]
I want to conduct a simulation " flow over 2d cylinder", does taichi_LBM3d provide function for the force evaluation.

**Comment by yjhp1016** [comment]
no, there is no module for this. you need to implement this yourself using
calculated pressure and velocity field

ztdepztdep ***@***.***> 于 2024年12月27日周五 下午11:13写道：
